src/methods/modular_programs/easy/q3/MeanNMedian.java

```diff
@@ -0,0 +1,34 @@
+package methods.modular_programs.easy.q3;
+
+/*
+Write a program that takes three initialized and declared integer variables and determines their mean and median.
+Note that the mean is the average of the three numbers and that the median is the middle number when the numbers are
+sorted from least to greatest. The program should have separate methods for computing both of these values, one called
+averageCalc and the other called medianCalc.
+Sample output:
+The average of the three integers is 13.0
+The median of the three integers is 5
+ */
+
+public class MeanNMedian {
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 2;
+        int c = 34;
+        System.out.println("The average of the three integers is " + averageCalc(a, b, c));
+        System.out.println("The median of the three integers is " + medianCalc(a, b, c));
+    }
+
+    public static double averageCalc(int a, int b, int c) {
```

Comment:
I feel like because we said to find the mean in the question, we should call it meanCalc instead of average calc